src/components/home.jsx

```diff
@@ -0,0 +1,82 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { NavLink } from 'react-router-dom';
+import { BsArrowRightCircle } from 'react-icons/bs';
+import './styles/home.css';
+import { fetchStocks, select, search } from '../redux/homeSlice';
+
+const Home = () => {
+  const stocks = useSelector((state) => state.stocks.stocks);
+  const loading = useSelector((state) => state.stocks.loading);
+  const searchTerm = useSelector((state) => state.stocks.searchTerm);
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    if (stocks.length === 0) {
+      dispatch(fetchStocks());
+    }
+  }, [dispatch, stocks.length]);
+
+  if (loading) {
+    return <h1>Loading...</h1>;
+  }
+
+  return (
+    <div className="stock-list-container">
+      <nav className="stock-list-navbar">
+        <div className="header-year-view">
+          <i className="fa-solid fa-arrow-up fa-rotate-270 fa-xs icon" />
+          <h2 className="stock-year">2023</h2>
+          <h1 className="stock-list-heading">more views</h1>
+          <i className="fa-solid fa-microphone fa-sm icon1" />
+          <i className="fa-solid fa-gear fa-sm icon2" />
+        </div>
```

Comment:
Well, you added the navbar as I mentioned in the previous review, which is a good step. However, there are still some adjustments needed:

- [ ] Currently, the elements are not laid out perfectly. The gaps and spacing need to be adjusted to match the design. The background color also requires some modifications. Please consider updating your code to align with the design. 

| Your | Design |
| --- | --- |
|  ![image](https://github.com/AbuTalha3/react-csp-metric-app/assets/55840999/a0a9960e-2a2c-4ef6-85c1-637c9469e27f)| ![image](https://github.com/AbuTalha3/react-csp-metric-app/assets/55840999/95fa805e-ad72-4812-8925-c172549ccd03) |

Making these adjustments will help ensure your navbar looks great and matches the intended design. Keep up the good work! 👍🎨💼